robottelo/helpers.py

```diff
@@ -232,6 +232,31 @@ def install_katello_ca(hostname=None):
             raise AssertionError('Failed to install the katello-ca rpm')
 
 
+def install_capsule_katello_ca(hostname=None, capsule=None):
```

Comment:
Move this code to vm.py's similar function under try block.

Reply:
@jyejare it's moved now